src/makefile.vc

```diff
@@ -4,146 +4,146 @@
-	pj_deriv.obj pj_ell_set.obj pj_ellps.obj pj_errno.obj \
-	pj_factors.obj pj_fwd.obj pj_init.obj pj_inv.obj \
+    aasincos.obj adjlon.obj bch2bps.obj bchgen.obj pj_gauss.obj \
+    biveval.obj dmstor.obj mk_cheby.obj pj_auth.obj \
+    pj_deriv.obj pj_ell_set.obj pj_ellps.obj pj_errno.obj \
+    pj_factors.obj pj_fwd.obj pj_init.obj pj_inv.obj \
     pj_fwd3d.obj pj_inv3d.obj \
-	pj_list.obj pj_malloc.obj pj_mlfn.obj pj_msfn.obj \
-	pj_open_lib.obj pj_param.obj pj_phi2.obj pj_pr_list.obj \
-	pj_qsfn.obj pj_strerrno.obj pj_tsfn.obj pj_units.obj \
-	pj_zpoly1.obj rtodms.obj vector1.obj pj_release.obj \
-	geocent.obj pj_transform.obj pj_datum_set.obj pj_datums.obj \
-	pj_apply_gridshift.obj pj_gc_reader.obj pj_gridcatalog.obj \
-	nad_cvt.obj nad_init.obj nad_intr.obj \
-	pj_utils.obj pj_gridlist.obj pj_gridinfo.obj \
-	proj_mdist.obj pj_mutex.obj pj_initcache.obj \
-	pj_ctx.obj pj_fileapi.obj pj_log.obj pj_apply_vgridshift.obj \
-	pj_strtod.obj pj_run_selftests.obj pj_generic_selftest.obj
+    pj_list.obj pj_malloc.obj pj_mlfn.obj pj_msfn.obj \
+    pj_open_lib.obj pj_param.obj pj_phi2.obj pj_pr_list.obj \
+    pj_qsfn.obj pj_strerrno.obj pj_tsfn.obj pj_units.obj \
+    pj_zpoly1.obj rtodms.obj vector1.obj pj_release.obj \
+    geocent.obj pj_transform.obj pj_datum_set.obj pj_datums.obj \
+    pj_apply_gridshift.obj pj_gc_reader.obj pj_gridcatalog.obj \
+    nad_cvt.obj nad_init.obj nad_intr.obj \
+    pj_utils.obj pj_gridlist.obj pj_gridinfo.obj \
+    proj_mdist.obj pj_mutex.obj pj_initcache.obj \
+    pj_ctx.obj pj_fileapi.obj pj_log.obj pj_apply_vgridshift.obj \
+    pj_strtod.obj pj_run_selftests.obj pj_generic_selftest.obj
 
 geodesic = geodesic.obj
-LIBOBJ	=	$(support) $(pseudo) $(azimuthal) $(conic) $(cylinder) $(misc) \
-	$(geodesic)
-PROJEXE_OBJ	= proj.obj gen_cheb.obj p_series.obj emess.obj
-CS2CSEXE_OBJ	= cs2cs.obj gen_cheb.obj p_series.obj emess.obj
-GEODEXE_OBJ	= geod.obj geod_set.obj geod_interface.obj emess.obj
+LIBOBJ  = $(support) $(pseudo) $(azimuthal) $(conic) $(cylinder) $(misc) \
+          $(geodesic)
+PROJEXE_OBJ  = proj.obj gen_cheb.obj p_series.obj emess.obj
+CS2CSEXE_OBJ = cs2cs.obj gen_cheb.obj p_series.obj emess.obj
+GEODEXE_OBJ  = geod.obj geod_set.obj geod_interface.obj emess.obj
 MULTISTRESSTEST_OBJ = multistresstest.obj
-PROJ_DLL 	= proj$(VERSION).dll
-PROJ_EXE    = proj.exe
-CS2CS_EXE   = cs2cs.exe
-GEOD_EXE    = geod.exe
-NAD2BIN_EXE = nad2bin.exe
+PROJ_DLL     = proj$(VERSION).dll
+PROJ_EXE     = proj.exe
+CS2CS_EXE    = cs2cs.exe
+GEOD_EXE     = geod.exe
+NAD2BIN_EXE  = nad2bin.exe
 MULTISTRESSTEST_EXE = multistresstest.exe
 
-CFLAGS	=	/nologo -I. -DPROJ_LIB=\"$(PROJ_LIB_DIR)\" \
-		-DHAVE_STRERROR=1 -DHAVE_LOCALECONV=1 $(OPTFLAGS)
+CFLAGS  = /nologo -I. -DPROJ_LIB=\"$(PROJ_LIB_DIR)\" \
+          -DHAVE_STRERROR=1 -DHAVE_LOCALECONV=1 $(OPTFLAGS)
 
-default:	all
+default:    all
 
 all: proj.lib $(PROJ_EXE) $(CS2CS_EXE) $(GEOD_EXE) $(NAD2BIN_EXE)
 
-proj.lib:	$(LIBOBJ)
-	if exist proj.lib del proj.lib
```

Comment:
> With Makefiles, it's not so much consistency that's the issue but the
requirement that the commands for a rule be indented with a TAB
character.

Ah, I was not aware of that! I've never made one of those from scratch, so never had a reason dig into all the details.

I am well aware that you can have different tab widths. That's why tabs are mostly bad in my opinion. Code looks different on different setups, which is fine if tabs are used consistently. That's rarely the case for the code in proj and leads to people indenting code with spaces to whatever tab width is in their particular editor. In my editor I have tabs showing up as "^I" to make them super obvious (I like to know what is going on, and have been fooled by stray tabs before...). This also makes mixed indentations look just a bit more horrible and scream at you to be fixed :)